logdb/logdb.go

```diff
@@ -59,7 +59,15 @@ func New(path string) (logDB *LogDB, err error) {
 		return nil, err
 	}
 
-	if _, err := wconn2.ExecContext(context.Background(), "pragma synchronous=off"); err != nil {
+	if _, err := wconn1.ExecContext(context.Background(), "PRAGMA journal_mode=WAL"); err != nil {
+		return nil, err
+	}
+
+	if _, err := wconn2.ExecContext(context.Background(), "PRAGMA journal_mode=MEMORY"); err != nil {
+		return nil, err
+	}
```

Comment:
I might be wrong on this, but If I remember properly even though we create 2 connections the `journal_mode` config applies to the DB (meaning that `MEMORY` would override `WAL`).

This is the source I found right now, I know, it is not crystal clear :( https://sqlite.org/pragma.html#pragma_journal_mode

Would it make sense to have a test for this?